assets/styles/app.css

```diff
@@ -1,3 +1,279 @@
-body {
-    background-color: lightgray;
+/* Navbar */
+.sub-header {
+  background: var(--main-color);
+  background: linear-gradient(0deg, var(--main-color) 0%, var(--secondary-color) 100%);
+  padding: 10px 0px;
+  border-bottom: 1px solid #eee;
 }
+
+.sub-header a:hover {
```

Comment:
Saw this class into the navbar html, remove it from navbar stylesheet if general behavior. Else remove it from here if it is only linked to layout navbar (and put it into navbar css file)